mpich-LEAP_15.spec

```
@@ -156,6 +156,7 @@ Requires:       libstdc++-devel
 %endif
 Requires:       %{name} = %{version}
 Requires:       daos-devel
+BuildRequires:  libfabric-devel
```

You want this to be a `Requires: libfabric-devel`:
```suggestion
Requires:  libfabric-devel
```